packages/workflow/src/Interfaces.ts

```diff
@@ -567,6 +567,7 @@ export interface INodeTypeDescription {
 		deactivate?: INodeHookDescription[];
 	};
 	webhooks?: IWebhookDescription[];
+	codex?: CodexCategories;
```

Comment:
@mutdmour If no contents, should `codex` point to an empty object or not be there at all? `codex` here set as optional, but the ticket says `codex` is always required.

Reply:
optional makes sense. 👍🏽